src/ParserVariables.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace secondtry
+{
+    class ParserVariables
+    {
+        public const string identifier = "configure";
+
+        public const string netidentifier = "network";
+        public const string devlistident = "network-dev";
+        public const string interlistident = "interface network-if";
+        
+        public const string voipidentifier = "voip";
+        public const string prosetlistident = "proxy-set";
+        public const string proiplistident = "proxy-ip";
+        
+        public const char tap = ' ';
+        public const string exit = "exit";
+        public const string activate = "activate";
+        public const string lz = "\n";
```

Comment:
bitte nimm hier als Value `System.Environment.NewLine`. Da deine Zeile unter bestimmten umständen nicht funktionieren wird. Siehe dazu (hier)[https://stackoverflow.com/questions/15433188/r-n-r-and-n-what-is-the-difference-between-them]. 

Somit wirst du unter Linux z.b. auch Erfolg haben da `NewLine` rücksicht auf das Environment nimmt